docs/INSTALL.md

```diff
@@ -69,6 +69,7 @@ used directly:
 ```
 dep ensure -v
 go install -v
+go install github.com/lightningnetwork/lnd/cmd/lncli
```

Comment:
Instead, we can modify the above to be: 
```
go install -v ./...
```

Reply:
Done. Also added it to **Updating** section.